src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -468,6 +472,15 @@ syncProgress = do
     <*> networkChainDifficulty
     <*> connectedPeers
 
+#ifdef DEV_MODE
+testResetAll :: WalletWebMode ssc m => m ()
+testResetAll = deleteAllKeys >> testReset
+  where
+    deleteAllKeys = do
+        keyNum <- length <$> getSecretKeys
+        sequence_ $ replicate keyNum $ deleteSecretKey 0
```

Comment:
In `cardano-sl-0.2` primary secret key is still stored as first in the list, so change it please to 
`replicate (keyNum - 1) $ deleteSecretKey 1`